webpack.config.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const LiveReloadPlugin = require('webpack-livereload-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const ASSET_PATH = process.env.ASSET_PATH || '/';
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'index.js',
+    publicPath: ASSET_PATH,
+    path: path.resolve(__dirname, 'dist')
+  },
+  module: {
+    rules: [
+      {
+        //   use: 'babel-loader',
```

Comment:
what is this comment for?

Reply:
For future going to ES6, I saw it in issues.